docs/source/daml-script/index.rst

```diff
@@ -235,15 +235,44 @@ translated to DAML script but there are a few things to keep in mind:
 #. Instead of specifying a ``scenario`` field in your ``daml.yaml``,
    you need to specify an ``init-script`` field. The initialization
    script is specified via ``Module:identifier`` for both fields.
-#. DAML script only supports the commands available on the ledger API
-   so you cannot call functions like ``fetch`` directly. This is
-   intentional. Your initialization scripts should not be able to
-   create transactions that a ledger client would not be able to
-   create. If you want to call methods not exposed via the Ledger API,
-   you can create a new template with a single choice
-   and call that via ``createAndExercise``.
-#. You need to replace calls to ``getParty x`` by
-   ``allocatePartyWithHint x (PartyIdHint x)``.
+#. In DAML script, ``submit`` and ``submitMustFail`` are limited to
+   the functionality provided by the ledger API: A list of independent
+   commands consisting of ``createCmd``, ``exerciseCmd``,
+   ``createAndExerciseCmd`` and ``exerciseByKeyCmd``. There are two
+   issues you might run into when migrating an existing scenario:
+
+   #. Your commands depend on each other, e.g., you use the result of
+      a ``create`` within a following command in the same
+      ``submit``. In this case, you have two options: If it is not
+      important that they are part of the translation, simply split
```

Comment:
Is that what you meant?
```suggestion
      important that they are part of a single transaction, split
```
Also, words like "simply" are a double-edged sword. People who don't immediately get what you mean might feel insulted.